eventiq_asb/broker.py

```diff
@@ -35,8 +37,9 @@ def __init__(
         self.batch_max_size = batch_max_size
         self._client: ServiceBusClient | None = None
         self._publisher: ServiceBusSender | None = None
-        self._receivers: dict[int, ServiceBusReceiver] = {}
         self._publisher_lock = anyio.Lock()
+        self.msgs_queues: dict[str, asyncio.Queue] = defaultdict(asyncio.Queue)
```

Comment:
max_size?

Reply:
queue is created when sender is started and maxsize is set to concurrency

Reply:
then it shouldn't be `defaultdict`